pyproject.toml

```diff
@@ -156,6 +156,11 @@ tracing-jaeger = ["opentelemetry-exporter-jaeger==1.14.0"]
 tracing-zipkin = ["opentelemetry-exporter-zipkin==1.14.0"]
 tracing-otlp = ["opentelemetry-exporter-otlp==1.14.0"]
 
+monitoring = [
+  "bentoml[monitoring-otlp]",
+]
```

Comment:
Hmm, idk if we should add this. I prefer explicitly declare external dependencies for these.

 The reason we have a `bentoml[tracing]` is for backward compatibility.